src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DbSnpIntegrationTest.java

```diff
@@ -0,0 +1,171 @@
+                    Allele.create("C", true),
+                    Allele.create("T")
+                },
+                {
+                    BaseFuncotatorArgumentCollection.FuncotatorReferenceVersionHg19,
+                    new SimpleInterval("chrY", 8551842, 8551842),
+                    "rs562075277",
+                    Allele.create("G", true),
+                    Allele.create("A")
+                },
+                {
+                    BaseFuncotatorArgumentCollection.FuncotatorReferenceVersionHg19,
+                    new SimpleInterval("chrM", 5005, 5005),
+                    "rs879008075",
+                    Allele.create("T", true),
+                    Allele.create("C")
+                },
+                // HG38 tests:
+                {
+                    BaseFuncotatorArgumentCollection.FuncotatorReferenceVersionHg38,
+                    new SimpleInterval("chr1", 84349785, 84349785),
+                    "rs17131617",
+                    Allele.create("T", true),
+                    Allele.create("C")
+                },
+                {
+                    BaseFuncotatorArgumentCollection.FuncotatorReferenceVersionHg38,
+                    new SimpleInterval("chrX", 80688070, 80688070),
+                    "rs3122407",
+                    Allele.create("T", true),
+                    Allele.create("C")
+                },
+                {
+                    BaseFuncotatorArgumentCollection.FuncotatorReferenceVersionHg38,
+                    new SimpleInterval("chrY", 13355944, 13355944),
+                    "rs2032654",
+                    Allele.create("A", true),
+                    Allele.create("G")
+                },
+                {
+                    BaseFuncotatorArgumentCollection.FuncotatorReferenceVersionHg38,
+                    new SimpleInterval("chrM", 5131, 5133),
+                    "rs199476116",
+                    Allele.create("TAA", true),
+                    Allele.create("T")
+                }
+        };
+    }
+
+    @Test(groups={"cloud"}, dataProvider = "provideFortestDbSnpDataSourceParsing")
+    public void testDbSnpDataSourceParsing( final String refVersion,
+                                            final Locatable interval,
+                                            final String expectedID,
+                                            final Allele expectedRefAllele,
+                                            final Allele expectedAltAllele) {
+        // 1 - Get the correct version of dbSNP from the funcotator data sources bucket:
+        final Path dbSnpFile = (refVersion.equals(BaseFuncotatorArgumentCollection.FuncotatorReferenceVersionHg38))
+                ? DB_SNP_HG38_FILE_PATH
+                : DB_SNP_HG19_FILE_PATH;
+
+        // 2 - Create a FeatureDataSource from the dbSNP VCF:
+        final FeatureDataSource<VariantContext> dbSnpDataSource = new FeatureDataSource<>(dbSnpFile.toUri().toString());
```

Comment:
Open this `FeatureDataSource` in a try-with-resources block to ensure that it gets closed.

Reply:
Yup.  Fixed!